JSTests/stress/big-int-strict-spec-to-this.js

```diff
@@ -18,6 +18,3 @@ for (let i = 0; i < 10000; i++) {
 for (let i = 0; i < 10000; i++) {
     assert(foo.apply(300) === "number");
 }
-
-assert(numberOfDFGCompiles(foo) > 1);
-
```

Comment:
Ditto.  Why remove?

Reply:
Refined instead of removing: `ToThis` is now simpler, so it takes only 1 DFG compilation.